### PR TITLE
Document possibility of installing ssg module with pip

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add few paragraphs to the developer guide
- fix a typo in the heading which directly follows the modified section

#### Rationale:

- improve documentation and transparency

### CODE REVIEW

File: docs/manual/developer/05_tools_and_utilities.md [hunk 1]
```diff
@@ -9,6 +9,27 @@ this easier: `.pyenv.sh`. To set `PYTHONPATH` correctly for the current
...
+```bash
+pip install git+https://github.com/ComplianceasCode/content
+```
+
+It is also possible to install an ssg library version associated with a certain release.
+This is recommended because the library is not stable and it can change unexpectedly when installing from master.
+This is an example command which installs the library associated with the 0.1.76 release:
+
+```bash
+pip install git+https://github.com/ComplianceasCode/content@v0.1.75
```
Comment (marcusburghardt) [inline docs/manual/developer/05_tools_and_utilities.md:24]:
```suggestion
pip install git+https://github.com/ComplianceasCode/content@v0.1.76
```

File: docs/manual/developer/05_tools_and_utilities.md [hunk 2]
```diff
@@ -9,6 +9,27 @@ this easier: `.pyenv.sh`. To set `PYTHONPATH` correctly for the current
...
+```
+
+The installed package is named `ssg`.
+Please note that the name of the package will very probably change in the future.
+Therefore, if you install the module in this way, please pay close attention to release notes.
```
Comment (marcusburghardt) [inline docs/manual/developer/05_tools_and_utilities.md:29]:
```suggestion
Therefore, if you install the module this way, please pay close attention to the release notes.
```

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
I have some suggestions about the wording.

**Comment by vojtapolasek** [comment]
@marcusburghardt  thank you, suggestions accepted.

File: docs/manual/developer/05_tools_and_utilities.md [hunk 3]
```diff
@@ -9,6 +9,29 @@ this easier: `.pyenv.sh`. To set `PYTHONPATH` correctly for the current
...
+It is also possible to install the module with `pip`.
+In this case it is recommended to install it within a Python virtual environment.
+Please note that this possibility was added after releasing the 0.1.75 version and until the release 0.1.76 is out it can only be installed from `master` branch.
+To install the ssg module currently present in the master branch, run the following command:
+
+```bash
+pip install git+https://github.com/ComplianceasCode/content
+```
+
+It is also possible to install an ssg library version associated with a certain release.
```
Comment (marcusburghardt) [inline docs/manual/developer/05_tools_and_utilities.md:21]:
```suggestion
It is also possible to install a ssg library version associated with a certain release.
```

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
I missed on minor typo in the first review. Sorry.

File: docs/manual/developer/05_tools_and_utilities.md (hunk 3 above)
Comment (marcusburghardt) [reply docs/manual/developer/05_tools_and_utilities.md:21]: All good so. TIL. : )

**Comment by marcusburghardt** [comment]
Failure in "Gate OL7" CI test is not related to this PR. It can be safely waived for this PR.

**Review by marcusburghardt — APPROVED** [review]
Thanks

**Pull request merged**